src/devices/third_reality.ts

```diff
@@ -344,6 +344,36 @@ const definitions: DefinitionWithExtend[] = [
         ],
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['3RWK0148Z'],
+        model: '3RWK0148Z',
+        vendor: 'Third Reality',
+        description: 'Smart Watering Kit',
+        fromZigbee: [fz.on_off, fz.battery],
+        toZigbee: [tz.on_off, tz.ignore_transition],
+        exposes: [e.switch(), e.battery(), e.battery_voltage()],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await endpoint.read('genPowerCfg', ['batteryPercentageRemaining']);
+            device.powerSource = 'Battery';
+            device.save();
+        },
+        extend: [
+            deviceAddCustomCluster('3rWateringSpecialCluster', {
```

Comment:
```suggestion
            battery(),
            onOff(),
            deviceAddCustomCluster('3rWateringSpecialCluster', {
```

After this `fromZigbee`/`toZigbee`/`exposes` and `configure` can be removed.